businessRules/ideas.js

```diff
@@ -38,34 +39,58 @@ function createIdeas(header, description, askedSum, userId) {
     createIdea(header, description, askedSum, userId);
 }
 
-function ideasStatusApproval(ideaId, approvalStatus) {
+function ideasStatusApproval(ideaId, isApproved) {
+    // Check if idea with this ideaId exists.
     const ideas = getAllIdeas();
     const idea = ideas.find((idea) => idea.id === ideaId);
 
-    if (approvalStatus === 'approved') {
-        updateIdea(ideaId, 'approve', 'approved');
+    if (!idea) {
+        throw new Error(`Error. No idea with such ID found.`);
+    }
+    // Idea exists.
+    let ideaStatus = '';
+    if (isApproved) {
```

Comment:
Try to simplify this `if else` statements.

Reply:
Updated and tested.